features/managing_order_message/send_email_to_customer.feature

```diff
@@ -0,0 +1,29 @@
+@order_message
```

Comment:
I think only one file will be enough. Just put this scenario in the first file.

Reply:
DONE